bindings/nodejs/lib/types/client/bridge/client.ts

```diff
@@ -357,8 +357,8 @@ export interface __NftIdToBech32Method__ {
     };
 }
 
-export interface __HexPublicKeyToBech32AddressMethod__ {
-    name: 'hexPublicKeyToBech32Address';
+export interface __PublicKeyHashMethod__ {
```

Comment:
Do we event need a bridge if there is no binding for it?

Reply:
No, forgot to remove